std/zip.d

```diff
@@ -156,8 +256,11 @@ final class ArchiveMember
     }
 
     /**
-     * Set the OS specific file attributes, as obtained by
-     * $(REF getAttributes, std,file) or $(REF DirEntry.attributes, std,file), for this archive member.
+     * Set the OS specific file attributes for this archive member.
+     *
+     * Params:
```

Comment:
In [.dscanner.ini](https://github.com/dlang/phobos/blob/master/.dscanner.ini) you can enable various checks by removing std.zip from the blacklists. This will help find more violations of the style guidelines and once addressed, ensure that future changes maintain them.

Reply:
Good to know! Thanks for that hint.